cli/src/main.rs

```diff
@@ -25,23 +25,23 @@ Ethereum ABI coder.
   Copyright 2016-2018 Parity Technologies (UK) Limited
 
 Usage:
-    ethabi encode function <abi-path> <function-name> [-p <param>]... [-l | --lenient]
-    ethabi encode params [-v <type> <param>]... [-l | --lenient]
-    ethabi decode function <abi-path> <function-name> <data>
-    ethabi decode params [-t <type>]... <data>
-    ethabi decode log <abi-path> <event-name-or-signature> [-l <topic>]... <data>
-    ethabi -h | --help
+	ethabi encode function <abi-path> <function-name> [-p <param>]... [-l | --lenient]
```

Comment:
One _last_ request: can we revert the spaces -> tabs conversions in the usage information here?